step/step.go

```diff
@@ -322,7 +322,7 @@ func (s XcodeTestRunner) getSimulatorForDestination(destinationSpecifier string)
 
 	simulatorDestination, err := destination.NewSimulator(destinationSpecifier)
 	if err != nil {
-		return simulator.Simulator{}, fmt.Errorf("invalid destination specifier: %v", err)
+		return simulator.Simulator{}, fmt.Errorf("invalid destination specifier: %s: %w", destinationSpecifier, err)
```

Comment:
Could standardize on `invalid destination specifier (%s): %w` or `invalid destination specifier '%s': %w` to avoid `:`